pkg/bindings/images/types_import_options.go

```diff
@@ -24,53 +19,27 @@ func (o *ImportOptions) Changed(fieldName string) bool {
 // ToParams
 func (o *ImportOptions) ToParams() (url.Values, error) {
 	params := url.Values{}
+
 	if o == nil {
 		return params, nil
 	}
-	json := jsoniter.ConfigCompatibleWithStandardLibrary
-	s := reflect.ValueOf(o)
-	if reflect.Ptr == s.Kind() {
-		s = s.Elem()
+
+	if o.Changes != nil {
+		panic("*** GENERATOR DOESN'T IMPLEMENT THIS YET ***")
```

Comment:
This crashes one of tests, however I believe it wasn't functioning correctly even before. `o.Changes` is `map[string]string` and therefore it was ignored before.

```go
lowerCaseKeys[iter.Key().Interface().(string)] = iter.Value().Interface().([]string)
```
If I understand it correcty `iter.Value().Interface().([]string)` will result in null if values happens to be just `string` instead of slice of strings.

Reply:
Oh I see `o.Changes` is not `map[string]string`, it's `*[]string`! I don't see any reason to use `*[]string` since `[]string` is already nullable, weird.

Reply:
The original code is removing pointer:
```golang
		if reflect.Ptr == f.Kind() {
			f = f.Elem()
		}
```

Reply:
Why is `map[string][]string` supported but `map[string]string` isn't?